src/main/resources/db/db.changelog-master.xml

```diff
@@ -608,4 +608,83 @@
         </addColumn>
     </changeSet>
 
+    <changeSet author="tomelliott (generated)" id="1569574616080-8">
```

Comment:
Okay, we don't want any of these changes. I have had this happen to me before - liquibase doesn't recognise the batch tables so tries to remove them.

Also, I can't see the one change we actually do want